dropwizard-jdbi/src/main/java/io/dropwizard/jdbi/DBIHealthCheck.java

```diff
@@ -1,23 +1,40 @@
 package io.dropwizard.jdbi;
 
 import com.codahale.metrics.health.HealthCheck;
+import io.dropwizard.util.Duration;
 import org.skife.jdbi.v2.DBI;
 import org.skife.jdbi.v2.Handle;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
 
 public class DBIHealthCheck extends HealthCheck {
     private final DBI dbi;
     private final String validationQuery;
-
-    public DBIHealthCheck(DBI dbi, String validationQuery) {
```

Comment:
This constructor is public, so users code could depend on it. For example, someone could create a custom *DBI* instance from a data source and add a health check.

Maybe we can retain it? We could use `MoreExecutors.newDirectExecutorService()` from Guava
and zero duration.